keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -402,129 +408,124 @@ type AdminNetworkPolicy struct {
+// Subject field. The traffic must match both ports and to.
 type AdminNetworkPolicyEgressRule struct {
-  // Name is an identifier for this rule. 
+  // Name is an identifier for this rule.
   // +optional
   Name         string
 
-  // Action specifies whether this rule must pass, allow or deny traffic. 
-  // Allow: allows the selected traffic 
+  // Action specifies whether this rule must pass, allow or deny traffic.
+  // Allow: allows the selected traffic
   // Deny: denies the selected traffic
-  // Pass: allows the selected traffic to skip and remaining positive priority AMP rules 
-  // and be delegated by K8's Network Policy. 
+  // Pass: allows the selected traffic to skip and remaining positive priority AMP rules
+  // and be delegated by K8's Network Policy.
   Action       AdminNetPolRuleAction
 
-  // L4Selector allows for matching on traffic based on L4 constructs. 
-  L4Selector   AdminNetworkPolicyL4Selector
+  // Ports allows for matching on traffic based on port and protocols.
+  Ports        AdminNetworkPolicyPorts
 
-  // List of destinations to which traffic will be allowed/denied/passed from the entities 
-  // selected by this AdminNetworkPolicyRule. Items in this list are combined using a logical OR 
-  // operation. If this field is empty, this rule matches no destinations. 
+  // List of destinations to which traffic will be allowed/denied/passed from the entities
+  // selected by this AdminNetworkPolicyRule. Items in this list are combined using a logical OR
+  // operation. If this field is empty, this rule matches no destinations.
   // If this field is present and contains at least one item, this rule
   // allows/denies/passes traffic to the defined AdminNetworkPolicyPeer(s)
   To           []AdminNetworkPolicyPeer
 }
 
-// AdminNetworkPolicyL4Selector handles selection of traffic for based on L4 
+// AdminNetworkPolicyPorts handles selection of traffic for based on L4
 // constructs. Exactly one of the fields must be defined.
-type AdminNetworkPolicyL4Selector struct { 
-  // AllPorts cannot be "false" when it is set 
-  // AllPorts allows the user to select all ports for all protocols, thus not 
+type AdminNetworkPolicyPorts struct {
+  // AllPorts cannot be "false" when it is set
+  // AllPorts allows the user to select all ports for all protocols, thus not
   // selecting traffic based on L4 principles.
-  // If "true" then all ports are selected for the all protocols
+  // If "true" then all ports are selected for the all protocols.
   AllPorts   *bool
 
-  // List of ports for outgoing traffic.
-  // Each item in this list is combined using a logical OR. If this field is
-  // empty or missing, this rule matches no ports (traffic not restricted by port).
-  // If this field is present and contains at least one item, then this rule 
-  // allows/denies/passes traffic only if the traffic matches at least one port in the list.
+  // List of destinattion ports for outgoing traffic.
```

Comment:
destination

Reply:
Wait actually this should be 

"The list of ports to `allow/deny/pass` traffic on"

Reply:
Done